.github/workflows/release.yml

```
@@ -122,3 +95,4 @@ jobs:
           prerelease: ${{ contains(github.ref_name, 'a') || contains(github.ref_name, 'b') || contains(github.ref_name, 'rc') }}
           generate_release_notes: true
           files: dist/*
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)